shell/gpu/gpu_surface_vulkan.cc

```diff
@@ -12,7 +12,7 @@ GPUSurfaceVulkan::GPUSurfaceVulkan(
     GPUSurfaceVulkanDelegate* delegate,
     std::unique_ptr<vulkan::VulkanNativeSurface> native_surface,
     bool render_to_surface)
-    : GPUSurfaceVulkan(nullptr,
+    : GPUSurfaceVulkan(nullptr /* GrDirectContext */,
```

Comment:
The proper way to comment these is with `/*foobar=*/`, where foobar is the name of the parameter according to the declaration.

https://google.github.io/styleguide/cppguide.html#Function_Argument_Comments